packages/app/src/app/overmind/namespaces/live/actions.ts

```diff
@@ -249,7 +249,7 @@ export const onViewRangeChanged: Action<UserViewRange> = (
 };
 
 export const onSelectionChanged: Action<UserSelection> = (
-  { state, effects },
+  { state, effects, actions },
```

Comment:
`actions` isn't used in `onSelectionChanged`

```suggestion
  { state, effects },
```